testnet/CHANGELOG.md

```
@@ -0,0 +1,384 @@
+  * Real-time: best bid/ask
+  * Every 100 ms: diff. depth
+  * Every 100 ms: partial book depth
+* For more information please refer to the [SBE Market Data Streams](sbe-market-data-streams.md).
+
+---
+
+### 2025-02-05
+
+* **Notice: These changes will be deployed starting at 7:00 UTC, and may take several hours to complete.**
+  **The following changes will apply to WebSocket Market Data Streams, User Data Streams, and the WebSocket API:**
+    * Our WebSocket services will send a ping frame **every 20 seconds** instead of 3 minutes.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)